hail/python/hail/experimental/vcf_combiner.py

```diff
@@ -168,3 +186,46 @@ def min_rep(locus, ref, alt):
 # ##INFO=<ID=RAW_MQ,Number=1,Type=Float>
 # ##INFO=<ID=ReadPosRankSum,Number=1,Type=Float>
 # ##INFO=<ID=VarDP,Number=1,Type=Integer>
+#
+# As of 2/6/19, the schema returned by the combiner is as follows:
+# ----------------------------------------
+# Global fields:
+#     None
+# ----------------------------------------
+# Column fields:
+#     's': str
+# ----------------------------------------
+# Row fields:
+#     'locus': locus<GRCh38>
+#     'alleles': array<str>
+#     'rsid': str
+#     'filters': set<str>
+#     'info': struct {
+#         DP: int64,
```

Comment:
Is this needed? The schema is different than the one above. I.e. The entry fields are `mt.entry.*`.

Reply:
If you are talking about `info.DP` in general, it's probably not necessary, and we could save some cycles/bytes not computing it, except in `summarize`.

My intention was to always have a "best effort" sparse matrix table, where the fields are as close as possible to the correct values that would be present in a VCF for the same samples.

What is present in the comments here is the output of `mt.describe()` after running a combiner sample pipeline.